guac-graphql.md

```diff
@@ -94,10 +94,18 @@ utility that can take a collection of files and ingest them into the GUAC graph.
 In your original window, run:
 
 ```bash
-bin/guacone collect files ../guac-data/docs/
+bin/guacone collect --add-vuln-on-ingest --add-eol-on-ingest --add-license-on-ingest files ../guac-data/docs/
 ```
 
-This can take a minute or two.
+This can take up to several minutes.
+
+{: .note }
+
+The `--add*` flags above will cause GUAC to query external services for
+additional data while ingesting the files. This will slow down the ingestion
```

Comment:
we can add as we are rate limited, hence the slow down on ingestion time.

Reply:
Good point. I'll add something like "Due to service rate limits and processing time, this will..."